lib/components/artist/related_artists/related_artist.js

```diff
@@ -15,11 +15,9 @@ class RelatedArtist extends React.Component {
     const {height, width} = Dimensions.get('window');
     const imageWidth = (width - 60)/2;
     const imageAspectRatio = 1.5;
-    console.log(imageWidth, width);
     return (
-      <View style={{ width: imageWidth }}>
-        <ImageView style={{ height: imageWidth/1.5 }}
-                   aspectRatio={1.5}
+      <View style={{paddingBottom: 30, width: imageWidth}}>
+        <ImageView style={{ width: imageWidth, height: imageWidth/imageAspectRatio }}
```

Comment:
Oh, I thought you had fixed dimensions, but I see now that this was a mistake on my part, as clearly you wanted it to be based on the view’s width. So using the aspectRatio prop would have made more sense then, as now you just need to repeat the native code we have in the image view… sorry about that. It’s not a blocker, we can just fix that up next time we go through this file.